docs/source/docker/advanced/active_learning.rst

```diff
@@ -1,7 +1,83 @@
 .. _ref-docker-active-learning:
 
-Active learning
-==============================================
+Active Learning
+===============
+
+Lightly makes use of active learning scores to select the samples which will yield
+the biggest improvements of your machine learning model. The scores are calculated
+on-the-fly based on model predictions and provide the selection algorithm with feedback
+about the uncertainty of the model for the given sample. 
+
+Learn more about the concept of active learning scores:
+:ref:`lightly-active-learning-scorers`.
+
+Prerequisites
+--------------
+In order to do active learning with Lightly, you will need the following things:
+
+- The installed Lightly docker (see :ref:`ref-docker-setup`)
+- A dataset with a configured datasource (see :ref:`ref-docker-with-datasource-datapool`)
+- Your predictions uploaded to the datasource (see :ref:`ref-docker-datasource-predictions`)
+
+.. note::
+
+    The dataset does not need to be new! For example, an initial selection without
+    active learning can be used to train a model. The predictions from this model
+    can then be used to improve your dataset by adding new images to it through active learning.
+
+
+Selection
+-------------------------
+Once you have everything set up as described above, you can do an active learning
+iteration by specifying the following three things in your Lightly docker config:
+
+- `method`
+- `active_learning.task_name`
+- `active_learning.score_name`
+
+Here's an example of how to configure an active learning run:
+
+
+.. tabs::
+
+    .. tab:: Web App
+
+        **Trigger the Job**
+
+        To trigger a new job you can click on the schedule run button on the dataset
+        overview as shown in the screenshot below:
+
+        .. figure:: ../integration/images/schedule-compute-run.png
+
+        After clicking on the button you will see a wizard to configure the parameters
+        for the job.
+
+        .. figure:: ../integration/images/schedule-compute-run-config.png
+
+        In this example we have to set the `active_learning.task_name` parameter
+        in the docker config. Additionally, we set the `method` to `coral` which
+        simultaneously considers the diversity and the active learning scores of
+        the samples. All other settings are default values. The
+        resulting docker config should look like this:
+
+        .. literalinclude:: code_examples/active_learning_worker_config.txt
+            :caption: Docker Config
+            :language: javascript
+
+        The Lightly config remains unchanged.
+
+    .. tab:: Python Code
+
+        .. literalinclude:: code_examples/python_run_active_learning.py
+
+
+
```

Comment:
Maybe make it more clear that the rest of the guide starting here is only for an older workflow and not recommended. E.g. that users know the guide actually stops here